File: wlxx-services/wlxx-system/wlxx-system-provider/src/main/java/com/xzg/wlxx/system/service/OrgService.java
package com.xzg.wlxx.system.service;

import com.baomidou.mybatisplus.extension.service.IService;
import com.xzg.wlxx.system.client.entity.po.OrgPo;
import com.xzg.wlxx.system.client.entity.vo.OrgVo;

import java.util.List;

public interface OrgService extends IService<OrgPo> {

    Boolean add(OrgPo org);

    List<OrgVo> list4Tree(Long rootId);
}

File: wlxx-services/wlxx-system/wlxx-system-provider/src/main/kotlin/com/xzg/wlxx/system/service/OrgService.kt
package com.xzg.wlxx.system.service

import com.baomidou.mybatisplus.extension.service.IService
import com.xzg.wlxx.system.client.entity.po.OrgPo
import com.xzg.wlxx.system.client.entity.vo.OrgVo

interface OrgService : IService<OrgPo?> {
    fun add(org: OrgPo): Boolean
    fun list4Tree(rootId: Long?): List<OrgVo?>
}
